packages/esm-patient-notes-app/src/notes/visit-notes-form.test.tsx

```diff
@@ -46,40 +51,36 @@ jest.mock('@openmrs/esm-framework', () => {
 });
 
 jest.mock('./visit-notes.resource', () => ({
-  fetchDiagnosisByName: jest.fn(),
+  fetchConceptDiagnosisByName: jest.fn(),
   fetchLocationByUuid: jest.fn(),
   fetchProviderByUuid: jest.fn(),
   saveVisitNote: jest.fn(),
 }));
 
 describe('Visit notes form: ', () => {
-  beforeEach(() => {
-    mockFetchLocationByUuid.mockResolvedValue(mockFetchLocationByUuidResponse);
-    mockFetchProviderByUuid.mockResolvedValue(mockFetchProviderByUuidResponse);
-    mockFetchDiagnosisByName.mockReturnValue(of(diagnosisSearchResponse.results));
-    mockUseConfig.mockReturnValue(ConfigMock);
-    mockUseSession.mockReturnValue(mockSessionDataResponse);
-  });
+  test('renders the visit notes form with all the relevant fields and values', () => {
+    mockFetchDiagnosisByName.mockReturnValue(of([]));
 
-  it('renders the visit notes form with all the relevant fields and values', () => {
     renderVisitNotesForm();
 
     expect(screen.getByRole('textbox', { name: /Visit date/i })).toBeInTheDocument();
     expect(screen.getByRole('textbox', { name: /Write an additional note/i })).toBeInTheDocument();
-    expect(screen.getByRole('group', { name: /Search for a diagnosis/i })).toBeInTheDocument();
+    expect(screen.getByRole('search', { name: /Enter Primary diagnosis/i })).toBeInTheDocument();
+    expect(screen.getByRole('search', { name: /Enter Secondary diagnosis/i })).toBeInTheDocument();
     expect(screen.getByRole('group', { name: /Add an image to this visit/i })).toBeInTheDocument();
-    expect(screen.getByRole('search', { name: /Enter diagnoses/i })).toBeInTheDocument();
     expect(screen.getByRole('button', { name: /Add image/i })).toBeInTheDocument();
     expect(screen.getByRole('button', { name: /Discard/i })).toBeInTheDocument();
     expect(screen.getByRole('button', { name: /Save and close/i })).toBeInTheDocument();
   });
 
-  it('typing in the diagnosis search input triggers a search', () => {
+  test('typing in the diagnosis search input triggers a search', () => {
```

Comment:
Hello @ibacher,
I am having trouble with most of the tests on this file. But if you could help me solve this one I belive I can make the other work :)